apps/cowswap-frontend/src/pages/error/AnySwapAffectedUsers/useIsAnySwapAffectedUser.ts

```diff
@@ -36,7 +36,7 @@ export default function useIsAnySwapAffectedUser() {
     AFFECTED_TOKENS,
     ERC20_INTERFACE,
     'allowance',
-    [account || ZERO_ADDRESS, ANYSWAP_V4_CONTRACT],
+    account ? [account || ZERO_ADDRESS, ANYSWAP_V4_CONTRACT] : undefined,
```

Comment:
Then you don't need the `ZERO_ADDRESS` part

```suggestion
    account ? [account, ANYSWAP_V4_CONTRACT] : undefined,
```